email_notification/validators.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from django.core.exceptions import ValidationError
+from sentry_sdk import capture_message
+
+from reservations.models import Reservation
+
+MAX_HTML_FILE_SIZE = 150000
+
+
+def html_file_validator(value):
+    ext = os.path.splitext(value.name)[1]
```

Comment:
Handy! Though, that seems to be a wrapper around libmagic C library which means even more dependencies. I have a feeling this might be a bit overkill for this use-case since it is just a sanity check that tries to avoid simple user errors 😄

Reply:
Yup, I agree the overkillness 👍